config/configauth/configauth_test.go

```diff
@@ -15,6 +15,11 @@ import (
 
 var mockID = component.MustNewID("mock")
 
+func TestNewDefaultAuthentication(t *testing.T) {
+	auth := NewDefaultAuthentication()
+	assert.NotNil(t, auth)
```

Comment:
Lets require that `AuthenticatorID` not be set.

Reply:
how do I do this?
tried setting it to an empty string but it keeps throwing an error.

Reply:
Something like `assert.Empty(t, auth)`

Reply:
Thank you
There are no errors again
Will push the commit now